x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/action/TransportStopRollupAction.java

```diff
@@ -52,12 +57,52 @@ protected void taskOperation(StopRollupJobAction.Request request,
                                  ActionListener<StopRollupJobAction.Response> listener) {
         if (jobTask.getConfig().getId().equals(request.getId())) {
             jobTask.stop(listener);
```

Comment:
we should not call the `listener` if `waitForStopped` is set.

Reply:
urgh yes, that would have been bad...